README.rst

```diff
@@ -45,7 +45,8 @@ The following configuration values are available:
   Defaults to enabled.
 - ``listenbrainz/token``: Your `Listenbrainz user token <https://listenbrainz.org/profile/>`_
 - ``listenbrainz/url``: The URL of the API of the Listenbrainz instance to record listens to (default: api.listenbrainz.org)
-
+- ``listenbrainz/import_playlists``: Whether to import Listenbrainz playlists (default: ``false``)
+- ``listenbrainz/periodic_playlists_update``: Enable periodic import of Listenbrainz playlists (default: ``true``)
```

Comment:
```suggestion
- ``listenbrainz/update_playlists``: Enable periodic import of Listenbrainz playlists (default: ``true``)
```
or something to keep the `verb_noun` order consistent.  let me know if you think there's a better name

Reply:
I understand and like the `verb_noun` paradigm. 

May I suggest `update_playlists_periodically`, or `monitor_playlists`?